config/logging.yaml

```diff
@@ -53,7 +55,7 @@
   :log_trace: false
   :level: info
   :type: file
-  :pattern: "%d [%c] [%.1l] %m\n"
+  :pattern: "%d %.8X{session} [%c] [%.1l] %m\n"
```

Comment:
I see @ohadlevy had the idea of using session id here: could we revisit this: I find this not that useful as having always request id there